src/colorizer/Dataset.ts

```diff
@@ -418,6 +419,14 @@ export default class Dataset {
       throw new Error("No features found in dataset. Is the dataset manifest file valid?");
     }
 
+    // Analytics reporting
+    triggerAnalyticsEvent(AnalyticsEvent.DATASET_LOAD, {
+      datasetWriterVersion: this.metadata.writerVersion || "N/A",
+      datasetTotalObjects: this.numObjects,
+      datasetFeatureCount: this.features.size,
+      datasetFrameCount: this.numberOfFrames,
```

Comment:
why not put manifest url here also?  wouldn't that make it easier?

Reply:
We absolutely could! I just wasn't sure if there were any concerns about data privacy in doing so (though we do collect URL params/path so technically we already have this anyways... :X)